hg-fast-export.py

```diff
@@ -161,7 +161,15 @@ def export_file_contents(ctx,manifest,files,hgtags,repourl,revnode,ignoreSub,enc
     if not hgtags and file == ".hgtags":
       sys.stderr.write('Skip %s\n' % (file))
       continue
+    # Skip .git files. They only get us in trouble.
+    if '.git' in file:
+      sys.stderr.write('Skip %s\n' % (file))
+      continue
     d=ctx.filectx(file).data()
+    # Skip files bigger than 3 MB
+    if len(d) > 3145728:
```

Comment:
I want to skip binary files since they should not be versioned.
I plan to further refine this code adding a lot of binary extensions and as a last resort the bigger than 3MB code.
I would like to later add some code to use Git LFS for those files, but I currently do not know if I'll have to do it with handwritten code or with a Python helper lib.